src/parser/jobs/sam/index.js

```diff
@@ -25,14 +25,25 @@ export default new Meta({
 
 	supportedPatches: {
 		from: '5.0',
```

Comment:
With the changes to Meikyou and Tsubame, logic is a bit diff now. The addition of Hagakure doesn't change anything on old logs but the other 2 skills do. I think it's probably a good idea to change the `from` version to `5.05` as well now, just to avoid bad suggestions on cooldown timing in 5.0 logs.

Reply:
Adjusted to not support 5.0